src/Enterspeed.Source.Sdk/Domain/Connection/EnterspeedConnection.cs

```diff
@@ -59,6 +60,10 @@ private void Connect()
             _httpClientConnection.DefaultRequestHeaders.Add("X-Api-Key", ApiKey);
             _httpClientConnection.DefaultRequestHeaders.Add("Accept", "application/json");
 
+            #if NETSTANDARD2_0_OR_GREATER || NET || NETCOREAPP2_0_OR_GREATER
+            _httpClientConnection.DefaultRequestHeaders.Add("X-Enterspeed-System", $".NET-SDK/{Assembly.GetExecutingAssembly()?.GetName().Version?.ToString()}");
```

Comment:
Could we come up with a better naming than `.NET-SDK`?
I'm thinking in terms of consistency if we add this other places.

Reply:
I'm more than happy, I just cannot think of a better name atm, any suggestions?

Reply:
I'm thinking of something like:
`sdk-dotnet`

It will fit well if/when SDKs in other languages:
`sdk-nodejs` etc.

Reply:
Changed